src/main/webapp/add_release.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+      <title>New Release</title>
+      <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+      <link rel="stylesheet" href="css/style.css">  
+  </head>
+
+   <body>
+
+        <div class="jumbotron">
+            <h1>New Release</h1>  
+        </div>  
+
+        <div class="row">
+            <div class="leftcolumn">
+                <div class="row">
+                    <div id="sdkleft">
+                        <label for="name">Release Name:</label>
+                        <input type="text" id="name" name="name"><br><br>
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div id="sdkleft">
+                        <label for="manager">Release Manager:</label>
+                        <input type="text" id="manager" name="manager"><br><br>
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div id="sdkleftlast">
+                        <label for="hotlist">Hotlists:</label>
+                        <input type="text" id="hotlist" name="hotlist"><br><br>
+                    </div>
+                </div>
+            </div>
+            <div class="rightcolumn">
+                <div class="row">
+                    <div id="sdklarge">
+                        <label for="coldfreeze">Cold Freeze:</label>
+                        <input type="text" id="coldfreeze" name="coldfreeze"><br><br>
+                        <label for="deadline">Launchcal Deadline:</label>
+                        <input type="text" id="deadline" name="deadline"><br><br>
+                        <label for="launch">Launch:</label>
+                        <input type="text" id="launch" name="launch"><br><br>
+                    </div>
+                </div>
+                
+                <div class="row">
+                    <div id="sdkright2">
+                        <p>Automatically Send Out Release Candidates at Cold Freeze</p>
```

Comment:
This is not an option to choose on the SDK. Its by default. You can remove this.

Reply:
These are options for a release and there was this on the sheet: https://screenshot.googleplex.com/eZ715HjCWmJ. PTAL.

Reply:
Should I remove this or keep it?

Reply:
You can keep it for now.